packages/plugin-fuse/src/actions/createToken.ts

```diff
@@ -118,11 +121,11 @@ export const createTokenAction = {
             {
                 user: "user",
                 content: {
-                    text: "Queue proposal 123 on the governor at 0x1234567890123456789012345678901234567890 on Ethereum",
-                    action: "QUEUE_PROPOSAL",
+                    text: "Create a token named 'MyToken' with symbol 'MTK' on Fuse.",
```

Comment:
should compare with plugin-avalanche examples. It should not need the ' characters.